static/styles/rewards/claim-table.css

```diff
@@ -149,6 +149,9 @@ table[data-make-claim-rendered] button.show > svg#claim-loader {
 table[data-make-claim-rendered] button.hide > svg#claim-icon {
   display: unset;
 }
+table[data-details-visible="true"][data-additional-data-size="large"] #mainTableBody {
+  transform: translate(-50%, 15%);
```

Comment:
You should use pixels.

Reply:
Kept it inline with the rest of the transforms ->  (`50%, npx`)

Reply:
Be mindful of the border thickness of 1px as well. I can't see clearly with the compression in the video but its sort of a common "gotcha" in these situations. Do not overlap borders.

Reply:
20px is as flush as I can get it